eNMS/forms/__init__.py

```diff
@@ -46,8 +48,23 @@ def __new__(cls, name, bases, attrs):
                     "integer": IntegerField,
                     "json": JsonField,
                     "string": StringField,
+                    "select": SelectField,
+                    "multiselect": SelectMultipleField,
                 }[values.get("type", "string")]
             form_kw = {"default": values["default"]} if "default" in values else {}
+            if field in [SelectField, SelectMultipleField]:
+                if "choices" in values:
+                    form_kw = {
+                        **form_kw,
+                        **{
+                            "choices": values["choices"],
+                            "render_kw": {
+                                "size": 1
+                                if field == SelectField
+                                else len(values["choices"])
```

Comment:
Why do we need that ?

Reply:
I added this based on my reading of the the WTForms documentation - "You'll need to specify the HTML `size`  attribute to the select field when rendering." (https://wtforms.readthedocs.io/en/2.3.x/fields/#wtforms.fields.SelectMultipleField) 

I will try it without and see - and provide feedback here in a few minutes.

Reply:
Absolutely had no effect- I should have tested without this earlier. Thank you and sorry for the confusion.

Reply:
Ok thanks, this is in `render_kw` so my guess is wtforms will make it the "html size" (size=...), which makes sense for a default html select list since they are rendered by default as a sort of textarea with a vertical size, but in eNMS these lists are rendered in javascript as a single line drop-down so that html size is ignored